src/actions/appActions.js

```diff
@@ -236,7 +242,7 @@ export const initAppAndRedirectAction = () => {
           },
         }),
       };
-    } else {
+    } else if(feautureOnboarding) {
```

Comment:
```suggestion
    } else if (feautureOnboarding) {
```

Do you have code linting enabled on your editor?

Reply:
yes